client/src/components/loginViewContainer.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from "react";
 import Login from "./LoginComponent/login";
-import style from "./LoginComponent/login.css";
 
 const LoginBody = () => {
   return <Login />;
```

Comment:
this seems redundant now that you only use it as a container, sorry I didn't catch this before
Unless this component will have more stuff in it

Reply:
I'm talking about importing Login and then exporting the same thing wrapped in LoginBody

Reply:
we resolved this in the next PR